src/main.js

```diff
@@ -64,12 +75,31 @@ function switchScreens(closingWindow, openingWindow){
 }
 
 function userInput() {
-  var newPoster = new Poster(userPosterImage.value, userPosterTitle.value, userPosterQuote.value);
-  posterImage.src = newPoster.imageURL;
-  posterTitle.innerHTML = newPoster.title;
-  posterQuote.innerHTML = newPoster.quote;
-  images.push(newPoster.imageURL);
-  titles.push(newPoster.title);
-  quotes.push(newPoster.quote);
-  savedPosters.push(newPoster);
-}
\ No newline at end of file
+  currentPoster = new Poster(userPosterImage.value, userPosterTitle.value, userPosterQuote.value);
+  posterImage.src = currentPoster.imageURL;
+  posterTitle.innerHTML = currentPoster.title;
+  posterQuote.innerHTML = currentPoster.quote;
+  images.push(currentPoster.imageURL);
+  titles.push(currentPoster.title);
+  quotes.push(currentPoster.quote);
+}
+
+function savePoster() {
```

Comment:
Attempt to keep your functions as SRP as possible. Currently, the `savePoster()` function is handling two responsibilities. This will also help keep your functions line count lower for a higher grade on your rubric.